File: tools/content-verify/__fixtures__/tracks/demo/problems/lc-9002-wrong-go/solution.py
from typing import Dict, List


class Solution:
    def groupByLength(self, words: List[str]) -> List[List[str]]:
        groups: Dict[int, List[str]] = {}
        for word in words:
            groups.setdefault(len(word), []).append(word)
        return list(groups.values())

File: tools/content-verify/__fixtures__/tracks/demo/problems/lc-9002-wrong-go/Solution.java
import java.util.*;

class Solution {
    public List<List<String>> groupByLength(String[] words) {
        Map<Integer, List<String>> groups = new LinkedHashMap<>();
        for (String word : words) {
            groups.computeIfAbsent(word.length(), length -> new ArrayList<>()).add(word);
        }
        return new ArrayList<>(groups.values());
    }
}
